client/auth/test/login.jsx

```diff
@@ -1,78 +0,0 @@
-/**
```

Comment:
One question -- it looks like this test file didn't make it anywhere else. Intended?